cfme/infrastructure/pxe.py

```diff
@@ -204,9 +206,11 @@ def exists(self):
         """
         Checks if the PXE server already exists
         """
-        dbs = self.impl()
```

Comment:
I think if we are going to not have these links that cast "visually" the implementation abstraction as a "nice" name, we possibly need to change the name of 'impl', maybe to 'endpnt', though I guess they are pretty similar. Thoughts?